src/player.cpp

```diff
@@ -3837,7 +3837,7 @@ void player::set_cat_level_rec(const std::string &sMut)
     if (!has_base_trait(sMut)) { //Skip base traits
         const auto &mdata = mutation_branch::get( sMut );
         for( auto &elem : mdata.category ) {
-            mutation_category_level[elem] += 8;
```

Comment:
I see. I guess those numbers should be reduced, too, right? So, for example, `p->mutation_category_level[primary] > 50` can become `p->mutation_category_level[primary] > 6`, since 6 * 8 = 48. This makes it clearer that 6 mutations in the category are required.